maps/Route44.asm

```diff
@@ -26,29 +26,29 @@ TrainerBirdKeeperVance1:
 	end_if_just_battled
 	opentext
 	checkflag ENGINE_VANCE
-	iftrue UnknownScript_0x19d86a
+	iftrue VanceWantsBattle
 	checkcellnum PHONE_BIRDKEEPER_VANCE
-	iftrue UnknownScript_0x19d8eb
+	iftrue NumberAcceptedM
```

Comment:
Don't use such generic labels; if Bird Keeper Vance has `NumberAcceptedM`, what will the many other phone-call trainers use? *Local* generic names are OK, like `.AskedAlready`, since they can be reused.

Reply:
The other phone call trainers use the exact same thing; all NumberAcceptedM does is `scall numberacceptedm`.  It doesn't build with simply .NumberAcceptedM because it's referenced by both Vance and Wilton.